docs/GETTING_STARTED.md

```diff
@@ -1,288 +1,271 @@
-```
-docker run -d -p 5000:5000 --restart=always --name registry ghcr.io/oci-playground/registry:v3.0.0-alpha.1
+```shell
+sudo make install
+# check soci can be found in PATH
+sudo soci --help
 ```
 
-Since this is just a locally run HTTP server we can interact with
-it using standard `curl` commands. To confirm this is running lets
-use the following commands:
+Many `soci` CLI commands need to be run as `sudo`, because the metadata is saved
+in directories that a non-root user often does not have access to.
 
-```
-docker ps
-```
+> This doc assumes SOCI binaries are installed into `PATH`. If not, please use
+> the full path of the binaries (e.g. `./out/soci`).
 
-which should yield a running container at 0.0.0.0:5000 for the ORAS image. Now
-to check that we can interact with the server run the following:
-```
-curl http://localhost:5000/v2/_catalog
-```
-This should return with `{"repositories":[]}` given that we haven't put anything
-in there yet
+## Push an image to your registry
 
-## Push an image to the registry using ctr
-First things first, let's choose our favorite image.  For the purpose of this
-document we will use the latest image of rabbitmq from DockerHub, aptly named
-`docker.io/library/rabbitmq:latest`.
+In this document we will use `rabbitmq` from DockerHub `docker.io/library/rabbitmq:latest`.
+We use [AWS ECR](https://docs.aws.amazon.com/AmazonECR/latest/userguide/getting-started-console.html)
+as the public registry for demonstration. Other OCI 1.0 compatible registries such
+as dockerhub should also work.
 
-To do this with ctr use the following command:
-```
-ctr i pull docker.io/library/rabbitmq:latest
-```
-To confirm this image (or the image you have chosen) is in containerd's local
-content store please run:
-```
-ctr i ls
-```
-and you should see the image.  Now we need to add an image reference it so that
-it can be added to our repository.
-```
-ctr i tag docker.io/library/rabbitmq:latest localhost:5000/rabbitmq:latest
+First let's pull the image from docker into containerd's data store, then (tag and)
+push it up to your registry:
+
+> The example assumes you have created an ECR repository called `rabbitmq` and
+> have credentials available to the AWS CLI. You just need to update `AWS_ACCOUNT` and `AWS_REGION`.
+>
+> If you are using a different registry, you will need to set `REGISTRY` and `REGISTRY_PASSWORD` appropriately
+> (and the `rabbitmq` repository is created or can be created automatically while pushing).
+>
+> The platform tag might be different depending on your machine.
+
+```shell
+export AWS_ACCOUNT=000000000000
+export AWS_REGION=us-east-1
+export REGISTRY_PASSWORD=$(aws ecr get-login-password --region $AWS_REGION)
+export REGISTRY=$AWS_ACCOUNT.dkr.ecr.$AWS_REGION.amazonaws.com
+sudo ctr i pull docker.io/library/rabbitmq:latest
+sudo ctr i tag docker.io/library/rabbitmq:latest $REGISTRY/rabbitmq:latest
+sudo ctr i push --user AWS:$REGISTRY_PASSWORD --platform linux/amd64 $REGISTRY/rabbitmq:latest
```

Comment:
Sorry, I should have caught this in the last comment. Can we also have `REGISTRY_USER=AWS` everywhere and note that you should also set it if you're using a different registry?

Reply:
sure, updated